Aug '23 membership update

cheeky-gorilla opened the pull request:
Member List
- Added 13 members
- Removed 3 members
- Changed "Team" where needed

Addresses and Weights
- As discussed with Trent, removed the table of addresses and weights because it can be seen onchain already (no need to also update documentation manually)

Quotes
- Added new member quotes (including updated "Team")

cheeky-gorilla committed:
Aug '23 membership update

Member List
- Added 13 members
- Removed 3 members
- Changed "Team" where needed

Addresses and Weights
- As discussed with Trent, removed the table of addresses and weights because it can be seen onchain already (no need to also update documentation manually)

Quotes
- Added new member quotes

jtraglia commented on docs/9-membership.md:
```diff
@@ -149,156 +157,15 @@ For transparency purposes, we are disclosing each Guild member's team, the name/
  | TXRX | [Anton Nashatyrev](https://github.com/Nashatyrev) | 
  | TXRX | [Mikhail Kalinin](https://github.com/mkalinin) | 
+ | ConsenSys DDS | [Chenyi Zhang](https://github.com/czhang-fm) | 
+ | ConsenSys DDS | [Roberto Saltini](https://github.com/saltiniroberto) | 
```
If this list is alphabetically, these two folks should be at the top.

Also, `ConsenSys` has [recently rebranded](https://github.com/Consensys) to `Consensys` (lower case "s").

cheeky-gorilla replied:
Good catch! Both updated.

jtraglia commented on docs/9-membership.md:
```diff
@@ -120,23 +123,28 @@ For transparency purposes, we are disclosing each Guild member's team, the name/
+ | Prysm | [Terence Tsao](https://github.com/terencechain) | 
+ | Stateless Consensus | [Guillaume Ballet](https://github.com/gballet) | 
+ | Stateless Consensus | [Ignacio Hagiopan](https://github.com/jsign) | 
+ | Stateless Consensus | [Kevaundray](https://github.com/kevaundray) | 
```
We could add Kev's last name:
```suggestion
 | Stateless Consensus | [Kevaundray Wedderburn](https://github.com/kevaundray) | 
```

jtraglia commented on docs/9-membership.md:
```diff
@@ -322,6 +190,7 @@ We asked members to share why they think the Protocol Guild is important for Eth
 **Core Contributor** - "I believe this experiment will result in a great precedent on how to keep the core values of the Ethereum protocol alive and relevant." <br><br>
 **Core Contributor** - "I think open source is really important today and is one of the pillars of the world today, and it is going to be more and more important in future, but as today a proper way to reward its developers is missing, so I hope that this experiment will create the basis for a wider approach that could improve the sustainability of open source development." <br><br>
 **Core Contributor** - "Incentive alignment is one of the core issues that impact the long term health of a software/network. I think the Protocol Guild helps solve a part of the incentive alignment problem by allowing members to gain exposure to the network they are developing. I feel like it will go a long way in helping keep and onboard new core developers, as well as ensure everyone has skin  ...
+**Core Contributor** - "it is an interesting concept to fund the Ethereum project, provides both funds and incentives/reward for core devs to do more, and that in my opinion set this open source community a part from the rest" <br><br>
```
Might want to capitalize the first word of their quote for consistency.

Also, a period at the end for consistency.

cheeky-gorilla replied:
Also updated! Thankee.

jtraglia reviewed (changes requested):
Looks good for the most part. Just a few nits.

cheeky-gorilla committed:
Justin's requested changes

cheeky-gorilla committed:
Add Kev's last name

Co-authored-by: Justin Traglia <95511699+jtraglia@users.noreply.github.com>

tvanepps commented on docs/9-membership.md:
```diff
@@ -151,154 +161,11 @@ For transparency purposes, we are disclosing each Guild member's team, the name/
 
 ## Addresses and Weights
-For member privacy, the order of weights below does not correspond to the names above. These addresses and weights will change over time as members change from full-time to part-time (or vice-versa), and as members are on/offboarded.
+Member addresses and weights can be seen in the [Split contract](https://app.0xsplits.xyz/accounts/0x84af3D5824F0390b9510440B6ABB5CC02BB68ea1/). For member privacy, the order of weights in the Split does not correspond to the names above.
```
> For member privacy, the order of weights in the Split does not correspond to the names above.

this can be removed, it made more sense when there was a list of addresses in the docs

tvanepps reviewed (changes requested):
only needs to remove some legacy text. other than that, looks good!

cheeky-gorilla committed:
Removed line as per Trent feedback

Removed: "For member privacy, the order of weights in the Split does not correspond to the names above."
